nncf/quantization/algorithms/min_max/backend.py

```diff
@@ -215,12 +215,13 @@ def get_target_point_shape(nncf_graph: NNCFGraph, node: NNCFNode, target_point:
 
     @staticmethod
     @abstractmethod
-    def get_weight_quantization_axes(node: NNCFNode, target_point: TargetPoint) -> Tuple[int, ...]:
+    def get_weight_quantization_axes(node: NNCFNode, target_point: TargetPoint, ndims: int) -> Tuple[int, ...]:
```

Comment:
Is it possible not to expand the signature of this function?

Reply:
No, need to have ndims to have detect axes, alternative to add graph to arguments

Reply:
yes, weight shape available in layer attributes, for module that contain weight attributes. 
As we discussed, need to rework layer attributes.